src/components/layout/Grid/Grid.tsx

```diff
@@ -55,19 +57,13 @@ const GridWrapper = styled.div<GridWrapperProps>`
   overflow: ${prop('$overflow')};
 `;
 const GridParent = styled.div<GridParentProps>(
```

Comment:
This is the same case as Cluster, there shouldn't be any need of using wrapper component

Reply:
I'd rather keep it as it's since it seems there are some gotchas related to content overflow, we also have some instances of the component that uses customized wrapper elements for some reason

Reply:
I think the gotchas were introduced by the need for a wrapper component, same for the custom wrapper utilization. I will check the cases in the platform.

Reply:
@julesbarr I went through the platform and all components that utilize wrapper component are doing it to set `overflow` to `visible` which will be what happens with the removal of the wrapper component by default so we are fine to remove it